tests/acceptance/features/lib/UpdatePasswordPage.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+/**
+ * ownCloud
+ *
+ * @author Kiran Parajuli <kiran@jankaritech.com>
+ * @copyright Copyright (c) 2020 Kiran Parajuli kiran@jankaritech.com
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU AFFERO GENERAL PUBLIC LICENSE
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU AFFERO GENERAL PUBLIC LICENSE for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public
+ * License along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace Page;
+
+use Behat\Mink\Session;
+use SensioLabs\Behat\PageObjectExtension\PageObject\Exception\ElementNotFoundException;
+
+/**
+ * PageObject for the password policy settings Page
+ *
+ */
+class UpdatePasswordPage extends OwncloudPage {
+	/**
+	 * @var string path
+	 */
+	protected $path = '/index.php/apps/password_policy/update_password?redirect_url=/coreAgain/index.php/apps/files/';
```

Comment:
Why protected? :thinking:

Reply:
IDK too...just copied and pasted it from `settingsPage` :thinking: Implemented in the same way for `$path` in other `pageObjects`.

Reply:
behat-page-object-extension `Page.php` has:
```
abstract class Page extends DocumentElement implements PageObject
{
    /**
     * @var string|null
     */
    protected $path = null;
...
```
so this is using the same variable scope as in the parent (grandparent) that it is overriding.

Reply:
https://www.php.net/manual/en/language.oop5.visibility.php - the parent abstract class does not want to make the value public, but expects the implementing class to be able to set, see and use it. So private also would not work. That leaves `protected`